demo/demo.md

```diff
@@ -21,38 +21,22 @@ The default component presents itself as a circle and default image placement.
 
 ## Component size options
 
-The auro-avatar component supports a `sm` and `md` avatar size.
+The auro-avatar component supports a `sm`, `md` and `lg` avatar types that determines the size of the image asset delivered.</br>
+<small>Note: The previous `sm` and `md` boolean attributes are deprecated.</small>
 
 <div class="exampleWrapper">
-  <auro-avatar md></auro-avatar>
-  <auro-avatar sm></auro-avatar>
+  <auro-avatar type="lg" code="boi"></auro-avatar>
+  <auro-avatar type="md" code="anc"></auro-avatar>
+  <auro-avatar type="sm" code="mke"></auro-avatar>
 </div>
 
 <auro-accordion lowProfile justifyRight>
   <span slot="trigger">See code</span>
 
   ```html
-  <auro-avatar md></auro-avatar>
-  <auro-avatar sm></auro-avatar>
-  ```
-
-</auro-accordion>
-
-## Airport Codes
-
-The auro-avatar supports auto generation of pre-defined images using the `code` attribute.
-
-<div class="exampleWrapper">
-  <auro-avatar code="anc"></auro-avatar>
-  <auro-avatar code="mke" sm></auro-avatar>
-</div>
-
-<auro-accordion lowProfile justifyRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-avatar code="anc"></auro-avatar>
-  <auro-avatar code="mke" sm></auro-avatar>
```

Comment:
`code` is still in the API documentation but examples removed here?

Reply:
this was addressed with the last update.